dozer-log/src/storage/local.rs

```diff
@@ -161,16 +161,22 @@ impl Storage for LocalStorage {
         let path = self.get_path(&key).await?;
         let file = tokio::fs::File::open(&path)
             .await
-            .map_err(|e| Error::FileSystem(path.clone(), e))?;
+            .map_err(|e| Error::FileSystem(path, e))?;
         let file = BufReader::new(file);
         Ok(ReaderStream::new(file).boxed())
     }
+
+    async fn delete_objects(&self, keys: Vec<String>) -> Result<(), Error> {
+        for key in keys {
+            let path = self.get_path(&key).await?;
+            std::fs::remove_file(&path).map_err(|e| Error::FileSystem(path, e))?;
```

Comment:
Sure. I was frightened by `tokio::fs::remove_file` spawning a thread, but I guess it's better than blocking the runtime.

Reply:
I think it won't spawn a thread, just mark the current tokio worker thread as potentially blocking.

Reply:
I read it from the [doc](https://docs.rs/tokio/latest/tokio/fs/fn.write.html).

> This operation is implemented by running the equivalent blocking operation on a separate thread pool using [spawn_blocking](https://docs.rs/tokio/latest/tokio/task/fn.spawn_blocking.html).

Looks like an extra thread?

Reply:
I read this in the module documentation:
> Tasks run by worker threads should not block, as this could delay servicing reactor events. Portable filesystem operations are blocking, however. This module offers adapters which use a blocking annotation to inform the runtime that a blocking operation is required. When necessary, this allows the runtime to convert the current thread from a worker to a backup thread, where blocking is acceptable.

But reading the implementation, it seems like it just always does `spawn_blocking`, which will usually spawn a thread.